src/bindings/intercept/env/README.md

```diff
@@ -1,5 +1,6 @@
 - infos = Information about elektrify getenv below
 - infos/author = Markus Raab <elektra@libelektra.org>
+- infos/status = stable intercept
```

Comment:
Ideally we bring the other README.md also to this format?

Reply:
Allright. I'm almost done finding out the authors.
I'm not sure about the author of the swig_python2 binding though. According to git log you created the files (e.g. CMakeLists.txt), but Manuel Mausz created the original python-legacy binding which got renamed to python2 right?

Reply:
Yes, Manuel Mausz (@manuelm) is the sole author of the python2 binding.